br/pkg/utils/db.go

```diff
@@ -16,8 +16,9 @@ import (
 
 var (
 	// check sql.DB and sql.Conn implement QueryExecutor and DBExecutor
-	_ DBExecutor = &sql.DB{}
-	_ DBExecutor = &sql.Conn{}
+	_                  DBExecutor = &sql.DB{}
+	_                  DBExecutor = &sql.Conn{}
+	logBackupTaskExist bool
```

Comment:
It's better to add a blank line  before `logBackupTaskExist bool` to avoid reformating previous two lines.